src/Test/Perf/.gitignore

```diff
@@ -2,5 +2,7 @@ artifacts/
 infra/bin/
 results/
 temp/
+DataBackup*/
+consumptionTempResults.xml
```

Comment:
These are the output files of CPC and I have a work item to move these data to `\\mlangfs1`. Once that moves out, I will remove these ignores